src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        Scanner sc = new Scanner(System.in);
+
+        String phrase = sc.nextLine();
+
+        while (!phrase.equals("bye")) {
+            if (phrase.equals("list")) {
+                Timetable.getTasks();
+                phrase = sc.nextLine();
+            } else if (phrase.contains("done")) {
+                phrase = phrase.replace("done ", "");
+                int taskNumber = Integer.parseInt(phrase);
+                Timetable.getThingsToDo().get(taskNumber - 1).finishTask();
+                phrase = sc.nextLine();
+            } else {
+                recordTask(phrase);
+                phrase = sc.nextLine();
+            }
+        }
+        printFarewellMessage();
+    }
+
+    static void printWelcomeMessage() {
+        String greeting = "Hi! I'm Herrick, your task manager." + "\n"
+                + "What would you like to add to your timetable?";
+        System.out.println(greeting);
+    }
+
+    static void printFarewellMessage() {
+        String farewell = "Bye. Hope you will complete everything for today!";
+        System.out.println(farewell);
+    }
+    static void recordTask(String phrase) {
+        System.out.println("Alright. I'll put it on the list.");
+
+        if (phrase.contains("todo")) {
+            Todo todo = new Todo(phrase.replace("todo ", ""));
+            Timetable.addTasks(todo);
+        } else if (phrase.contains("deadline")) {
+            phrase = phrase.replace("deadline ","");
+            String[] whatAndWhen = phrase.split(" /by ");
+            Deadline deadline = new Deadline(whatAndWhen[0], whatAndWhen[1]);
+            Timetable.addTasks(deadline);
+        } else if (phrase.contains("event")) {
+            phrase = phrase.replace("event ","");
+            String[] whatAndWhen = phrase.split(" /at ");
+            Event event = new Event(whatAndWhen[0], whatAndWhen[1]);
+            Timetable.addTasks(event);
+        }
+
+        if (Timetable.size() <= 1) {
+            System.out.println("For now, you have " + Timetable.size() + " task on the list");
```

Comment:
As mentioned above, the style of coding should be consistent where earlier u initialised a string first before printing out greetings but here u print the text out instantly so maybe itll be more consistent if u stuck to one style!

Also, is this if statement necessary as it is repeating the code?